server/controllers/chartsController/helpers.test.js

```diff
@@ -121,13 +122,39 @@ describe('chartsController - helpers', () => {
         ],
       })
 
-      expect(helpers.generateStudyTargetTotals(chart)).toEqual({
+      expect(helpers.generateStudyTargetTotals(chart, allowedStudies)).toEqual({
         MA: { count: 0, targetTotal: 20 },
         NY: { count: 0, targetTotal: 100 },
         Foo: { count: 0, targetTotal: 8 },
         Totals: { count: 0, targetTotal: 128 },
       })
     })
+    it('results with a targetTotal of undefined when a site does not include targetValues', () => {
```

Comment:
this is a confusing test description. Is this testing that the "totals" site will have an undefined targetTotal when a site does not have a target value?

Reply:
https://github.com/AMP-SCZ/dpdash/pull/221/commits/97d6295d9cb36f821ae3ee45edcf0d501ed06a80